src/app/components/auth/reset-password-confirm/reset-password-confirm.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
`Reset-password` and `Reset-password-confirm` component seem to be having many things in common. Do we really need two separate components for two closely-dependent functionalities?

Reply:
As both provide different functionality, therefore, I think we should have separate component for them.

Reply:
Tha's true, but I had noticed a lot of similarities in the code for both the components. That's why I brought this. But this is of less-priority, so we can take it in future PR's.